src/librustc/hir/intravisit.rs

```diff
@@ -490,7 +490,12 @@ pub fn walk_item<'v, V: Visitor<'v>>(visitor: &mut V, item: &'v Item) {
             visitor.visit_ty(typ);
             visitor.visit_generics(type_parameters)
         }
-        ItemKind::Existential(ExistTy { ref generics, ref bounds, impl_trait_fn: _ }) => {
+        ItemKind::Existential(ExistTy {
+            ref generics,
+            ref bounds,
+            impl_trait_fn: _,
```

Comment:
`..` seems slightly preferable and the pattern would fit on a single line again.

Reply:
I originally did this to make sure users adding to `ExistTy` would know to check if they needed to add a corresponding visit here, but I think new additions which require this seem pretty unlikely, so I'll switch.